frontend/src/v5/ui/controls/formSelect/formSelect.component.tsx

```diff
@@ -16,8 +16,9 @@
  */
 
 import React from 'react';
-import { Select, InputLabel, FormControl, SelectProps } from '@material-ui/core';
+import { InputLabel, FormControl, SelectProps } from '@material-ui/core';
 import { Controller } from 'react-hook-form';
+import { Select } from '@controls/select';
```

Comment:
We can get rid of  our Select control and just put those props as default props in the theme.

Reply:
Done